soda-core/src/soda_core/contracts/impl/contract_verification_impl.py

```diff
@@ -49,6 +52,28 @@
 logger: logging.Logger = soda_logger
 
 
+class ContractVerificationHandler:
+    @classmethod
+    def instance(cls, identifier: Optional[str] = None) -> Optional[ContractVerificationHandler]:
+        # TODO: replace with plugin extension mechanism
+        create_method: Callable[..., Optional[ContractVerificationHandler]] = Extensions.find_class_method(
```

Comment:
as mentioned above - I would leave the exception in the find_class_method and just catch it here